src/App.jsx

```diff
@@ -9,32 +11,25 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

Comment:
it's better to have the full tab here
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```